packages/cli/src/scripts/validate-json-all.ts

```diff
@@ -114,26 +116,26 @@ async function validate(schemaName: string, schemaInfo: SchemaInfoItem) {
       encoding: 'utf8',
     });
 
-    let data: any = null;
     try {
-      data = JSON.parse(contentAsString);
-    } catch (e) {
-      console.group();
-      console.error(chalk.bgRed.bold(`  ${fileName} cannot be parsed  \n`));
-      console.groupEnd();
-      return false;
-    }
+      const data: JSONObject = JSON.parse(contentAsString);
+      sortTimeSeriesInDataInPlace(data);
 
-    const { isValid, schemaErrors } = executeValidations(
-      validateFunction,
-      data,
-      schemaInfo
-    );
+      const { isValid, schemaErrors } = executeValidations(
+        validateFunction,
+        data,
+        schemaInfo
+      );
 
-    if (!isValid) {
+      if (!isValid) {
+        console.group();
+        console.error(schemaErrors);
+        console.error(chalk.bgRed.bold(`  ${fileName} is invalid  \n`));
+        console.groupEnd();
+        return false;
+      }
+    } catch (e) {
       console.group();
-      console.error(schemaErrors);
-      console.error(chalk.bgRed.bold(`  ${fileName} is invalid  \n`));
+      console.error(chalk.bgRed.bold(`  ${fileName} cannot be parsed  \n`));
```

Comment:
the catch can catch more errors than only the json parsing..
```suggestion
      console.error(e);
```

Reply:
nit; I think it's pretty rare that that stack trace gives you information that the message doesn't. Ans also we have a util for printing errors in red. So all these lines can be simplified to this IMO:

```
logError(`Failed to process file ${fileName}: ${err.message}`)
```